src/telemetry/backend-metrics.yaml

```
@@ -0,0 +1,236 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# For more information on configuring this file:
+# https://mozilla.github.io/glean/book/reference/yaml/metrics.html
+
+---
+$schema: moz://mozilla.org/schemas/glean/metrics/2-0-0
+
+page:
+  view:
+    type: event
+    description: |
+      A web page view event distinguishable by the page’s URL.
+    bugs:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1896222
+    data_reviews:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1896222
+    data_sensitivity:
+      - interaction
+    notification_emails:
+      - rhelmer@mozilla.com
+    expires: never
+    send_in_pings:
+      - monitor-events
```

I'd recommend using the built-in `events` ping (which requires no definition in a pings config file) rather than a custom ping.
```suggestion
      - events
```
I'm guessing you were following FxA's lead here, but we're now in the process of switching FxA's Glean setup to use the built-in `events` ping ([DENG-2407](https://mozilla-hub.atlassian.net/browse/DENG-2407)).

---

Yes, all the event metrics should be sent in the default `events` ping, and you don't need to define a `monitor-events` ping.